poker/src/components/LobbySetting/LobbySetting.tsx

```diff
@@ -73,14 +73,14 @@ const LobbySetting: FC = () => {
         <p>Scram master as player:</p>
         <Checkbox name={'master/player'} isChecked={settings.masterIsPlayer} onChange={handleChangeMasterOrPlayer} />
       </div>
-      <div className={styles.lobbyPage_setting_item}>
+      {/* <div className={styles.lobbyPage_setting_item}>
         <p>Changing card in round end:</p>
         <Checkbox
           name={'isChangeCardInRoundEnd'}
           isChecked={settings.isChangeCard}
           onChange={handleChangeCardInRoundEnd}
         />
```

Comment:
a lot of comments in code

Reply:
according to the task, the cards should be turned over after all participants have voted. if we keep up with the deadlines, then I will uncomment this code and use it. if we don't have time, I'll just delete it